tests/pybaseball/datasources/test_fangraphs.py

```diff
@@ -60,9 +60,7 @@ class TestDatasourceFangraphs:
     def test_get_table(self, sample_html, sample_processed_result):
         actual_result = fangraphs.get_fangraphs_tabular_data_from_html(
             sample_html,
-            '//table',
-            ["CS%"],
-            ["Team"]
+            '//table'
```

Comment:
Is it only ever something else in the testing framework? Can we default it to rg and have the `//table` only in the tests? wondering

Reply:
If so I can just adjust the HTML of the test to add that class.

Reply:
Or not because it's actual FG HTML, so it's already there. lol